contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
```

Comment:
By running `python run.py lib/DefineBot/DefineBot.py --config-file ../.zuliprc-local` from the `contrib-bots` directory.